submissions/yanyshpolska/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,182 @@
+                  src="./img/icons/blogs.png"
+                  alt=""
+                />
+                <span class="popup__item_name">Blogs</span>
+              </a>
+              <a href="#" class="popup__item">
+                <img
+                  class="popup__item_image"
+                  src="./img/icons/chat.png"
+                  alt=""
+                />
+                <span class="popup__item_name">Chat</span>
+              </a>
+              <a href="#" class="popup__item">
+                <img
+                  class="popup__item_image"
+                  src="./img/icons/calendar.png"
+                  alt=""
+                />
+                <span class="popup__item_name">Calendar</span>
+              </a>
+              <input
+                class="popup__item_checkbox_input visually-hidden"
+                type="checkbox"
+                id="popup__checkbox"
+              />
+              <div class="popup__item_more">
+                <div class="popup__list">
+                  <a href="#" class="popup__item">
+                    <img
+                      class="popup__item_image"
+                      src="./img/icons/classroom.png"
+                      alt=""
+                    />
+                    <span class="popup__item_name">Classroom</span>
+                  </a>
+                  <a href="#" class="popup__item">
+                    <img
+                      class="popup__item_image"
+                      src="./img/icons/mars.png"
+                      alt=""
+                    />
+                    <span class="popup__item_name">Mars</span>
+                  </a>
+                  <a href="#" class="popup__item">
+                    <img
+                      class="popup__item_image"
+                      src="./img/icons/apps.png"
+                      alt=""
+                    />
+                    <span class="popup__item_name">Applications</span>
+                  </a>
+                  <a href="#" class="popup__item">
+                    <img
+                      class="popup__item_image"
+                      src="./img/icons/collection.png"
+                      alt=""
+                    />
+                    <span class="popup__item_name">Collection</span>
+                  </a>
+                </div>
+              </div>
+              <label for="popup__checkbox" class="popup__item_checkbox_label">
+                <span class="popup__item_checkbox_more">More</span>
+              </label>
+            </div>
+          </div>
+        </div>
+        <a class="menu__item_picture menu__item" href="#">
+          <img
+            class="menu__item_picture_img"
+            src="./img/bell.png"
+            alt="Notifications"
+          />
+        </a>
+        <a class="menu__item_picture menu__item" href="#">
+          <img class="menu__item_picture_img" src="./img/logo.jpg" alt="User" />
```

Comment:
https://www.w3schools.com/tags/att_img_alt.asp

Reply:
Changed the "alt" value from "User" to "Profile".

Reply:
`The text should explain where the link goes if the image is inside an <a> element` 
So, it should be "Go to Profile page"

Reply:
Made corresponding changes to `alt`s